.github/workflows/ci.yml

```diff
@@ -25,22 +32,30 @@ jobs:
         uses: ./.github/actions/init-scala-env
 
       - name: Compile Scala Project
+        env:
+          SPARK_VERSION: ${{ matrix.spark_version }}
         run: make compile-scala
 
       - name: Test Scala Project
         # python/* branches are not supposed to change scala code, trust them
         if: ${{ !startsWith(github.event.inputs.from_branch, 'python/') }}
-        run: make test-scala
+        env:
+          SPARK_VERSION: ${{ matrix.spark_version }}
+        run:
+          ulimit -c unlimited
```

Comment:
@pomadchin, without this, test suite started to crash for Spark versions <3.4 after I added frameless 0.14.1. Any idea why?

Reply:
Nope, that's a good observation though